resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Constants.scala

```diff
@@ -64,7 +64,9 @@ private[spark] object Constants {
   val ENV_SPARK_CONF_DIR = "SPARK_CONF_DIR"
   val ENV_SPARK_USER = "SPARK_USER"
   // Spark app configs for containers
-  val SPARK_CONF_VOLUME = "spark-conf-volume"
+  val SPARK_CONF_VOL_DRV = "spark-conf-vol-drv"
```

Comment:
If you don't mind, 
`SPARK_CONF_VOL_DRV` -> `SPARK_CONF_VOLUME_DRIVER`?
`spark-conf-vol-drv` -> `spark-conf-volume-driver`?